site/pages/api/hootsuite-callback.api.ts

```diff
@@ -101,10 +101,21 @@ const hootsuiteCallback = async (req: NextApiRequest, res: NextApiResponse) => {
 
         const userId: string = userDetails.data.id;
 
+        let refreshTokens = null;
+        try {
+            refreshTokens = (await getParametersByPath(`/social/${session.orgId}/hootsuite`)) ?? null;
+        } catch (e) {}
+
+        const refreshTokenExists = refreshTokens?.Parameters?.find((rt) => rt.Name?.includes(userId));
+
+        if (refreshTokenExists) {
+            throw new Error("Refresh token already exists");
```

Comment:
Why are we throwing an error?

Reply:
If we have received a new token from Hootsuite then this should be updated in SSM - if we use an older refresh token then the api's will no longer work

Reply:
this is if a different use hits connect then another token is made as they become the creator of a new refresh token - it does not overwrite. The connection should only happen once to hootsuite

Reply:
if the refresh token for this other user was already available and the user has clicked the connect to hootsuite button again - in this case an error would be thrown but the refresh token would be lost

Reply:
it shouldn't be available in the first place there should only be one

Reply:
as discussed the data displayed in the table should be for a user but the latest refresh token should also be saved in SSM

Reply:
done